src/core/thread/topology.hpp

```diff
@@ -351,6 +351,14 @@ class Neighbor : public InstanceLocatorInit
      */
     void SetRloc16(uint16_t aRloc16) { mRloc16 = aRloc16; }
 
+    /**
+     * This method indicates whether or not it is a valid Thread 1.1 neighbor.
+     *
+     * @returns TRUE if it is a valid Thread 1.1 neighbor, FALSE otherwise.
+     *
+     */
+    bool IsThreadVersion1_1(void) const { return mState != kStateInvalid && mVersion == OT_THREAD_VERSION_1_1; }
```

Comment:
General point here to discuss and consider our naming style and how to handle numbers and whether we want to use underscoer `_` in camel-case style names.
- I personally favor avoiding `_` (as much as possible) in any camel case style name.
- We can consider `IsThreadVersion1d1`  ("one dot one") or `IsThreadVersion1p1` ("one point one").
  -  We use something similar for "2.4GHz" in `OPENTHREAD_CONFIG_RADIO_2P4GHZ_OQPSK_SUPPORT`.

Thoughts?

Reply:
I am fine with `IsThreadVersion1p1`. Updated.